src/axolotl/utils/__init__.py

```diff
@@ -3,10 +3,18 @@
 """
 import importlib.util
 
+import torch
+from packaging import version
+
 
 def is_mlflow_available():
     return importlib.util.find_spec("mlflow") is not None
 
 
 def is_comet_available():
     return importlib.util.find_spec("comet_ml") is not None
+
+
+def is_torch_min(version_string: str):
+    torch_version = version.parse(torch.__version__)
```

Comment:
I found that `torch.__version__` can equal `2.3.1+cu121`. Does this parser handle it?

Reply:
yes